src/Features/Core/Portable/MoveToNamespace/MoveToNamespaceOptionsResult.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+
+namespace Microsoft.CodeAnalysis.MoveToNamespace
+{
+    internal class MoveToNamespaceOptionsResult
```

Comment:
Will your dialog have a `Preview Changes` checkbox?

Reply:
Not with current design